collections/_article/release-candidate-godot-4-2-rc-2.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Release candidate: Godot 4.2 RC 2"
+excerpt: "The first release candidate brought another crop of regression reports, which we've now fixed!"
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/release-candidate-godot-4-2-rc-2.webp
+image_caption_title: "2D platformer prototype"
+image_caption_description: "An open source game by Securas"
+date: 2023-11-24 17:00:00
+---
+
+Since our first release candidate for Godot 4.2 a week ago, a number of regressions have been reported by new testers who were starting to test using Godot 4.2 in production. So we've been busy this week triaging, testing, debugging, and fixing regressions to reach a point where we're confident that we're ready for the stable release.
+
+That's where this second release candidate comes in, polished as much as we can and ready to be trialed on all your Godot 4.1 projects, hopefully as a smooth upgrade! If you notice anything off that's not already listed under the [known issues](#known-issues) below, please make sure to report it quickly so we can keep track of it in our release planning.
+
+This release candidate includes a number of big changes that warrant particular attention:
+
+- Many users have been running into spurious `slot >= slot_max` errors in their release exports in Godot 4.0 and 4.1, typically around changing scenes or otherwise manipulating nodes in the scene tree. We finally managed to debug and fix it properly ([GH-85280](https://github.com/godotengine/godot/pull/85280)), which should solve a lot of issues, including crashes on Windows. This change should find its way in a 4.1.x maintenance release too.
```

Comment:
```suggestion
- Many users have been running into spurious `slot >= slot_max` errors in their release exports in Godot 4.0 and 4.1, typically around changing scenes or otherwise manipulating nodes in the scene tree. We finally managed to debug and fix this properly ([GH-85280](https://github.com/godotengine/godot/pull/85280)), which should solve a lot of issues, including crashes on Windows. This change should find its way in a 4.1.x maintenance release too.
```